godot-macros/src/derive_godot_class.rs

```diff
@@ -231,6 +325,56 @@ fn make_godot_init_impl(class_name: &Ident, fields: Fields) -> TokenStream {
     }
 }
 
+fn make_godot_properties_impl(
+    class_name: &Ident,
+    properties: Vec<PropertyInfoAttribute>,
+) -> TokenStream {
+    let property_info_tokens: Vec<TokenStream> = properties
+        .into_iter()
+        .map(|property_info: PropertyInfoAttribute| -> TokenStream {
+            use std::str::FromStr;
+            let name = proc_macro2::Literal::from_str(&property_info.name).unwrap();
+            let getter = proc_macro2::Literal::from_str(&property_info.getter).unwrap();
+            let setter = proc_macro2::Literal::from_str(&property_info.setter).unwrap();
```

Comment:
The reason I went with `="my_setter"` rather than `=my_setter` is because there's no guarantee that users will expose methods to godot using their rust name (though I guess this library currently enforces that).
I've dealt with enough binding libraries that I've seen all kinds of conflicting naming requirements and would recommend building to support divorcing rust's function name form the godot-exposed name. I could totally see going with the function identifier for now and adding an option for string values later once `#[func]` gets support for renaming.

What do you want me to go with for this first PR?

Reply:
Oh, the reason I couldn't go with Literal::string() was (IIRC) that the property_info fields include the enclosing `"` characters. So `#getter` would expand to `"\"my_getter\""` rather than `"my_getter"`.
Is there a better way to handle this?

Reply:
> The reason I went with ="my_setter" rather than =my_setter is because there's no guarantee that users will expose methods to godot using their rust name (though I guess this library currently enforces that).

That's a valid point indeed. It depends a bit how this is used -- is it referring to a Rust function, or just the _name_ of a registered function? Maybe some example code would be helpful 😉 

>  Is there a better way to handle this?

I'm not aware of one, feel free to write a utility function for this if it occurs multiple times.